server/routes/routerRequestHandlers.test.ts

```diff
@@ -371,4 +377,144 @@ describe('routerRequestHandlers', () => {
       expect(next).not.toHaveBeenCalled()
     })
   })
+
+  describe('retrievePrisonerSummaryIfNotInSession', () => {
```

Comment:
All of these tests were moved from the now deleted `server/routes/overview/prisonerSummaryRequestHandler.test.ts`